pkgs/development/cuda-modules/setup-hooks/auto-fix-elf-files.sh

```diff
@@ -2,7 +2,14 @@
 # List all dynamically linked ELF files in the outputs and apply a generic fix
 # action provided as a parameter (currently used to add the CUDA or the
 # cuda_compat driver to the runpath of binaries)
-echo "Sourcing cuda/fix-elf-files.sh"
+
+[[ -n ${autoFixElfFiles_Once-} ]] && return
+declare -g autoFixElfFiles_Once=1
+
+echo "Sourcing auto-fix-elf-files.sh"
+
+declare -ag elfPrependRunpaths
+declare -ag elfAddRunpaths
```

Comment:
This doesn't help, `elfAddRunpaths` set in derivations (without `__structuredAttrs`) are still interpreted as strings with spaces

Reply:
This is annoying. Can we force in some way such derivations to set `__structuredAttrs`, or find another way to parse `elfAddRunpaths`?

Reply:
The annoying bit is that this is wrong and results in a singleton array